docs/src/main/paradox/java/http/common/uri-parsing.md

```diff
@@ -0,0 +1,157 @@
+## The URI model
+
+Akka HTTP offers its own specialised `URI` model class which is tuned for both performance and idiomatic usage within
```

Comment:
We should try to follow the spelling and casing of the actual classes. So, it would be `Uri` and `HttpRequest`, below.